src/exchange.ts

```diff
@@ -0,0 +1,101 @@
+import BN from "bn.js";
+import { TokenInfo } from "../src";
+import { BatchExchangeInstance } from "../types/truffle-typings";
+
+const MAXU32 = new BN(2).pow(new BN(32)).sub(new BN(1));
+
+export async function fetchTokenInfoFromExchange(
```

Comment:
nit: If these are no longer utility methods, but intended to be used outside of the `dex-contracts` package, they should probably include some documentation.

Reply:
Yes, I agree. Do you have any opinion about whether they should or should not be utility methods?

Reply:
I think they are useful exchange methods to be re-exported.

Reply:
Done!